lib/msal-common/src/cache/entities/IdTokenEntity.ts

```diff
@@ -53,6 +57,27 @@ export class IdTokenEntity extends CredentialEntity {
     }
 
     /**
+     * fetches idToken from cache if present
+     * @param request
+     */
+    static readIdTokenFromCache(cacheManager: CacheManager, clientId: string, account: AccountInfo, realm?: string): IdTokenEntity {
```

Comment:
Maybe these should be helpers on CacheManager instead?

Reply:
Yeah I considered that as well. I initially thought this made more organizational sense but after thinking more on it I'm inclined to agree with you

Reply:
Done.